modules/cluster-logging-exported-fields-kubernetes.adoc

```diff
@@ -1,60 +1,385 @@
+
+
+
+
+==== kubernetes.labels.deployment
+
+The deployment associated with this Kubernetes object
+
+[horizontal]
+Data type:: keyword
+
+Example value:: `logging-kibana-3`
+
+
+
+
+==== kubernetes.labels.deploymentconfig
+
+The deployment configuration associated with this Kubernetes object
+
+[horizontal]
+Data type:: keyword
+
+Example value:: `logging-kibana`
+
+
+
+
+
+
+==== kubernetes.labels.component
+
+The component associated with this Kubernetes object
+
+[horizontal]
+Data type:: keyword
+
+Example value:: `kibana`
+
+
+
+
+==== kubernetes.labels.provider
+
+The provider associated with this Kubernetes object
+
+[horizontal]
+Data type:: keyword
+
+Example value:: `openshift`
+
+
+
+
+
+=== kubernetes.event
+
+The Kubernetes event obtained from the Kubernetes master API. The event is a whole JSON object nested under the `kubernetes` field. This event description loosely follows `type Event` in link:https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.21/#event-v1-core[Event v1 core].
+
+[horizontal]
+Data type:: group
+
+
+
+
+==== kubernetes.event.verb
+
+The type of event: `ADDED`, `MODIFIED`, or `DELETED`
+
+[horizontal]
+Data type:: keyword
+
+Example value:: `ADDED`
+
+
+
+
+==== kubernetes.event.metadata
+
+Information related to the location and time of the event creation
+
+[horizontal]
+Data type:: group
+
+
+
+
+===== kubernetes.event.metadata.name
+
+The name of the object that triggered the event creation
+
+[horizontal]
+Data type:: keyword
+
+Example value:: `java-mainclass-1.14d888a4cfc24890`
+
+
+
+
+===== kubernetes.event.metadata.namespace
+
+The name of the namespace that induced the event. This is different from `namespace_name`.
+// TBD
+
+[horizontal]
+Data type:: keyword
+
+Example value:: `default`
+
+
+
+
+===== kubernetes.event.metadata.selfLink
+
+A link to the event
+
+[horizontal]
+Data type:: keyword
+
+Example value:: `/api/v1/namespaces/javaj/events/java-mainclass-1.14d888a4cfc24890`
+
+
+
+
+===== kubernetes.event.metadata.uid
+
+The unique ID of the event
+
+[horizontal]
+Data type:: keyword
+
+Example value:: `d828ac69-7b58-11e7-9cf5-5254002f560c`
+
+
+
+
+===== kubernetes.event.metadata.resourceVersion
+
+A string that identifies the server's internal version of the event. Clients can use this string to determine when objects have changed.
+
+[horizontal]
+Data type:: integer
+
+Example value:: `311987`
+
+
+
+
+==== kubernetes.event.involvedObject
+
+A description of the object involved in the event creation
+
+[horizontal]
+Data type:: group
+
+
+
+
+===== kubernetes.event.involvedObject.kind
+
+The type of object
+
+[horizontal]
+Data type:: keyword
+
+Example value:: `ReplicationController`
+
+
+
+
+===== kubernetes.event.involvedObject.namespace
+
+The name of the namespace in which the object triggered the event. If a pod did not trigger the event, then the namespace is not the `kubernetes.namespace_name`. This is the case for every event in the eventrouter namespace.
```

Comment:
@alanconway  Similarly, can you help me revise the last phrase, in italics, here. The original in the yaml file reads, "description: The name of the namespace in which the object triggered the event. If a pod did not trigger the event, then the namespace is not the `kubernetes.namespace_name`. _This is the case for every event eventrouter's namespace._"

Reply:
Same deal. Drop the event section till I can confirm we need it and verify the info is current and correct. With the viaq/doc site we can easily make doc updates available even between releases.

Reply:
Marked private.